packages/api/mocks/ProductQuery.ts

```diff
@@ -229,15 +229,15 @@ export const productSearchFetch = {
     correction: {
       misspelled: true,
     },
-    fuzzy: '0',
+    fuzzy: 'auto',
     operator: 'and',
     translated: false,
     pagination: {
       count: 1,
       current: {
         index: 1,
         proxyUrl:
-          'search/trade-policy/1?page=1&count=1&query=sku:64953394&sort=&locale=en-US&hide-unavailable-items=false&operator=and&fuzzy=0',
+          'search/trade-policy/1?page=1&count=1&query=sku:64953394&sort=&fuzzy=auto&operator=and',
```

Comment:
we don't need `locale=en-US&hide-unavailable-items=false` parameter for this test, right?

Reply:
I don't think so, the tests are passing.
<img width="657" alt="Screenshot 2024-09-02 at 19 40 17" src="https://github.com/user-attachments/assets/bf2278ca-6ece-473d-998d-92cbe2b9dfec">